dm_cli/package_tree_from_zip.py

```diff
@@ -88,4 +88,9 @@ def package_tree_from_zip(
             update=True,
         )
 
+        # Make sure to replace relative references in _meta_
+        root_package.meta = replace_relative_references(
+            "_meta_", root_package.meta, dependencies, data_source_id, file_path=root_package.path()
+        )
```

Comment:
Shoudnt we call "replace_relative_references" on the whole package, not just "_meta_". In case the "type" in package also uses an alias.

Reply:
never mind. The package's "type" seem to be hardcoded.